Xamarin.Forms.Core/BaseExportRendererAttribute.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+
+namespace Xamarin.Forms.Core
+{
+	[AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
+	public abstract class BaseExportRendererAttribute : HandlerAttribute
+	{
+		protected int MajorVersion { get; set; }
+
+		#region MinimumSdkVersion
+		int _minimumSdkVersion = int.MinValue;
+		public int MinimumSdkVersion
+		{
+			get { return _minimumSdkVersion; }
+			set
+			{
+				if (_minimumSdkVersion == value)
+					return;
+
+				_minimumSdkVersion = value;
+
+				if (_minimumSdkVersion > MaximumSdkVersion)
+					throw new ArgumentException("Minimum SDK version must be less than or equal to the maximum SDK version.");
+			}
+		}
+		#endregion
+
+		#region MaximumSdkVersion
+		int _maximumSdkVersion = int.MaxValue;
+		public int MaximumSdkVersion
+		{
+			get { return _maximumSdkVersion; }
+			set
+			{
+				if (_maximumSdkVersion == value)
+					return;
+
+				_maximumSdkVersion = value;
+
+				if (_maximumSdkVersion < MinimumSdkVersion)
+					throw new ArgumentException("Maximum SDK version must be greater than or equal to the minimum SDK version.");
+			}
+		}
+		#endregion
+
+		protected BaseExportRendererAttribute(Type handler, Type target) : base(handler, target)
+		{
+		}
+
+		public override bool ShouldRegister()
+		{
+			if (MinimumSdkVersion <= MajorVersion && MajorVersion <= MaximumSdkVersion)
+				return base.ShouldRegister();
```

Comment:
I think, it does not need to base call
I think, it does not need to base call. . 
Because, 
base class can't change. 
and HandlerAttribute.ShouldRegister always returning true.

Reply:
Why do you assume base class can't change? While it is unlikely that it will change, if it happens for whatever reason, then not calling base here would ignore base completely. `ShouldRegister` should perhaps be abstract (or an interface method) and sub-classes should implement it. Right now, base `ShouldRegister` is not doing anything other than returning a boolean result, which is mostly useless, so abstracting it out could be a better choice. What does the team think?